docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade your organization to paid
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I upgrade my organization to paid?
+
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+
```

Comment:
It'd also be good to add links to:

* [Database usage pricing](https://www.featurebase.com/pricing){:target="_blank:}
* [Available database sizes](https://docs.featurebase.com/docs/cloud/cloud-databases/cloud-db-shape/)

Ideally the pricing and sizing would be on one page which I'll investigate how to do later :)

Reply:
good call